corehq/apps/ota/views.py

```diff
@@ -32,6 +35,59 @@ def restore(request, domain, app_id=None):
     return get_restore_response(domain, couch_user, app_id, **get_restore_params(request))
 
 
+@json_error
+@login_or_digest_or_basic_or_apikey()
+def search(request, domain):
+    """
+    Accepts search criteria as GET params, e.g. "https://www.commcarehq.org/a/domain/phone/search/?a=b&c=d"
+    Returns results as a fixture with the same structure as a casedb instance.
+    """
+    criteria = dict(request.GET)
+    try:
+        case_type = criteria.pop('case_type')
+    except KeyError:
+        return HttpResponse('Search request must specify case type', status=400)
+
+    search_es = CaseSearchES()
+    search_es = search_es.domain(domain)
+    search_es = search_es.case_type(case_type)
+    fuzzies = get_fuzzy_properties(domain, case_type)  # TODO: define
+    for key, value in criteria.items():
+        search_es = search_es.case_property_query(key, value, fuzzy=(key in fuzzies))
+    results = search_es.values()
+    fixtures = CaseDBFixture([CommCareCase.wrap(case) for case in results])  # TODO: Wait, and rebase.
```

Comment:
Looks good, however, this won't work with SQL cases. (check discussion here: https://github.com/dimagi/commcare-hq/pull/10981#discussion_r58247742).